src/core/FormModel.ts

```diff
@@ -14,24 +14,30 @@ export class FormModel<Values> {
     [K in keyof Values]: FormField<Values[K]>;
   } = {} as { [K in keyof Values]: FormField<Values[K]> };
 
-  setFormField<T extends keyof Values>(
+  setFormField<T extends keyof Values & (string | symbol)>(
```

Comment:
It strange and it is same as following:

```suggestion
  setFormField<T extends string | symbol>(
```

Reply:
`setFormField<T extends string | symbol>` will miss the code suggestion for the key of `Values`